gcloud/bigtable/happybase/connection.py

```diff
@@ -97,6 +100,16 @@ def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, timeout=None,
             raise ValueError('Protocol cannot be set for gcloud '
                              'HappyBase module')
 
+        if table_prefix is not None:
+            if not isinstance(table_prefix, six.string_types):
```

Comment:
Do we actually want to allow anything other than native `str` here?  ISTM that allowing `unicode` on Pythonn2 is likely to blow up later on.